### PR TITLE
Fix/7740 json schema bg se

### PR DESCRIPTION
**Pull request by ArturFriesen**
## Description
Adjust schema to support QR codes from Bulgaria and Sweden.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7740

IMG_C7B6EAD4C8F9-1

### CODE REVIEW

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Ressources/CertificateSchema.json [hunk 1]
```diff
@@ -28,30 +28,31 @@
       "title": "Date of birth",
       "description": "Date of Birth of the person addressed in the DGC. ISO 8601 date format restricted to range 1900-2099",
       "type": "string",
-      "pattern": "(19|20)\\d{2}-\\d{2}-\\d{2}",
+      "__format": "date",
+      "pattern": "(19|20)\\d{2}-\\d{2}-\\d{2}(T00:00:00)?",
```
Comment (nickguendling) [inline src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Ressources/CertificateSchema.json:32]: I think our Formatter in `HealthCertificate.swift` using `.justLocalDateFormatter` might fail with "T00:00:00", have you tried this?

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Ressources/CertificateSchema.json (hunk 1 above)
Comment (mlenkeit) [reply src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Ressources/CertificateSchema.json:32]: You can try https://github.com/eu-digital-green-certificates/dgc-testdata/blob/main/BG/png/BG-1.png to test it. It should display correctly. If it crashes, then the formatting must be adjusted.

Comment (ArturFriesen) [reply src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Ressources/CertificateSchema.json:32]: Yes, I tested it. It works.

File: src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Ressources/CertificateSchema.json [hunk 2]
```diff
@@ -170,7 +171,8 @@
         "dt": {
           "description": "Date of Vaccination",
           "type": "string",
-          "pattern": "(19|20)\\d{2}-\\d{2}-\\d{2}",
+          "__format": "date",
+          "pattern": "(19|20)\\d{2}-\\d{2}-\\d{2}(T00:00:00)?",
```
[the remark this answers is not in the document]
Comment (ArturFriesen) [reply src/xcode/ENA/HealthCertificateToolkit/Sources/HealthCertificateToolkit/Ressources/CertificateSchema.json:175]: works

**Review by nickguendling — APPROVED** [review]

**Review by freshking — APPROVED** [review]

**Pull request merged**